Create Executive vote - April 17, 2025.md (#2888)
makerdao/community: governance/votes/Executive vote - April 17, 2025.md

date: 2025-04-17T00:00:00.000Z
address: "$spell_address"

---
# [Executive Proposal] SP-BEAM Initialization, Sky Token Rewards Rebalance, Set Aave Lido Market (Prime Market) DDM DC to 0, SBE Changes, Launch Project Funding, Spark-Aave Revenue Share Payment, AD Compensation, Atlas Core Development Payments, Spark Proxy Spell - April 17, 2025

The Governance Facilitators, Sidestream, and Dewiz have placed an executive proposal into the voting system. MKR/SKY Holders should vote for this proposal if they support the following alterations to the Sky Protocol.

If you are new to voting in the Sky Protocol, please see the [voting guide](https://manual.makerdao.com/governance/voting-in-makerdao/on-chain-governance) to learn how voting works.

---

## Executive Summary

If this executive proposal passes, the following **actions** will occur within the Sky Protocol:

- The Stability Parameter Bounded External Access Module (SP-BEAM) will be initialized as described below.

> - The Sky Token Rewards (SKY to USDS) APY will be adjusted.

Reviewer: ```suggestion
- The Sky Token Rewards (SKY to USDS) Annual Percentage Yield (APY) will be adjusted.
```

      - `cfg.ilks.step`: 400 basis points
- Add SP-BEAM to the [Chainlog](https://chainlog.sky.money) as `MCD_SPBEAM` at [0x36B072ed8AFE665E3Aa6DaBa79Decbec63752b22](https://etherscan.io/address/0x36B072ed8AFE665E3Aa6DaBa79Decbec63752b22).
- Add SPBEAMMom to the [Chainlog](https://chainlog.sky.money) as `SPBEAM_MOM` at [0xf0C6e6Ec8B367cC483A411e595D3Ba0a816d37D0](https://etherscan.io/address/0xf0C6e6Ec8B367cC483A411e595D3Ba0a816d37D0).

### Sky Token Rewards Rebalance

- **Authorization**: [Ecosystem Authorization](https://forum.sky.money/t/sky-token-rewards-update-april-17-spell/26254/5)
- **Proposal**: [Forum Post](https://forum.sky.money/t/sky-token-rewards-update-april-17-spell/26254)

If this executive proposal passes, then the [Sky Token Rewards (SKY to USDS)](https://info.sky.money/rewards/0x0650caf159c5a49f711e8169d4336ecb9b950275) APY will be adjusted to match the SSR by changing the rewards distribution rate through the following actions:

- Yank `MCD_VEST_SKY` vest with ID 1.
- Claim the remaining tokens from the old DssVest by calling `VestedRewardsDistribution.distribute()`.
- Create a new `MCD_VEST_SKY` stream with the following parameters:

>   - Rewards Distribution Rate: 160,000,000

Reviewer 2: ```suggestion
  - Rewards Distribution Rate: 160 million
```

  - Rewards Distribution Cap: 176,000,000
  - Start Date relative to the spell cast time:
    - `vestBgn`: `block.timestamp`

>   - End Date is set one year after Start Date defined by duration:

Reviewer 2: ```suggestion
  - End Date relative to the spell cast time:
```

    - `vestTau`: 182 days
    - `vestFin`: `block.timestamp` + 182 days
- File the new stream ID on `REWARDS_DIST_USDS_SKY`.

### Set Aave Lido Market (Prime Market) Direct Deposit Module Debt Ceiling to 0

- **Authorization**: [Ecosystem Authorization](https://forum.sky.money/t/spark-aave-lido-market-usds-allocation/25311/25)
- **Proposal**: [Forum post](https://forum.sky.money/t/spark-aave-lido-market-usds-allocation/25311/24)

> If this executive proposal passes, then the Aave Lido Market (Prime Market) Direct Deposit Module (DDM) Debt Ceiling (DC) will be set to 0 by executing the following actions:

Reviewer 2: ```suggestion
If this executive proposal passes, then the Aave Lido Market (Prime Market) Direct [Deposit Module (DDM)](https://sky-atlas.powerhouse.io/A.3.8.1.1.2.4_Debt_Ceiling_Instant_Access_Module_(DC_IAM)/071d42e3-8a21-4401-852e-0b52c49768bb%7C57eaf45219bea3b430c2) [Debt Ceiling (DC)](https://sky-atlas.powerhouse.io/A.3.8.1.1.2.4.1_Maximum_Debt_Ceiling_(line)/6f1a913d-9436-4b70-816b-e317672737d6%7C57eaf45219bea3b430c268bb) will be set to 0 by executing the following actions:
```

- Remove `DIRECT-SPK-AAVE-LIDO-USDS` from autoline.
- Set `DIRECT-SPK-AAVE-LIDO-USDS` DC to zero.

### Smart Burn Engine Changes

- **Authorization**: [Ecosystem Authorization](https://forum.sky.money/t/smart-burn-engine-parameter-update-april-17-spell/26253/2)
- **Proposal**: [Forum post](https://forum.sky.money/t/smart-burn-engine-parameter-update-april-17-spell/26253)

> If this executive proposal passes, then the SBE parameter below will be updated as follows:

Reviewer 3: ```suggestion
If this executive proposal passes, then the SBE [parameter](https://sky-atlas.powerhouse.io/A.3.6.2_Surplus_Buffer_Splitter_Parameters/122f2ff0-8d73-80f8-9a2a-d221794f73f5|57ea2c54) below will be updated as follows:
```

- Increase `Splitter.hop` by 493 seconds from 1,235 seconds to **1,728 seconds**.

### Launch Project Funding

- **Authorization**: [Atlas A.5.6](https://sky-atlas.powerhouse.io/A.5.6_Launch_Project/1f433d9d-7cdb-406f-b7e8-f9bc4855eb77%7C8d5a)
- **Proposal**: [Forum post](https://forum.sky.money/t/utilization-of-the-launch-project-under-the-accessibility-scope/21468/30)

> If this executive proposal passes, then the following Launch Project funding distributions will be made:

Reviewer 3: ```suggestion
If this executive proposal passes, then the following [Launch Project](https://sky-atlas.powerhouse.io/A.5.5_Launch_Project/1f433d9d-7cdb-406f-b7e8-f9bc4855eb77|8d5a) funding distributions will be made:
```

| Delegate    | Amount (USDS) | Address                                                                                                               |
|-------------|--------------:|-----------------------------------------------------------------------------------------------------------------------|
| BLUE        | 4,000         | [0xb6C09680D822F162449cdFB8248a7D3FC26Ec9Bf](https://etherscan.io/address/0xb6C09680D822F162449cdFB8248a7D3FC26Ec9Bf) |
| Bonapublica | 4,000         | [0x167c1a762B08D7e78dbF8f24e5C3f1Ab415021D3](https://etherscan.io/address/0x167c1a762B08D7e78dbF8f24e5C3f1Ab415021D3) |
| Cloaky      | 4,000         | [0x9244F47D70587Fa2329B89B6f503022b63Ad54A5](https://etherscan.io/address/0x9244F47D70587Fa2329B89B6f503022b63Ad54A5) |
| JuliaChang  | 4,000         | [0x252abAEe2F4f4b8D39E5F12b163eDFb7fac7AED7](https://etherscan.io/address/0x252abAEe2F4f4b8D39E5F12b163eDFb7fac7AED7) |
| WBC         | 4,000         | [0xeBcE83e491947aDB1396Ee7E55d3c81414fB0D47](https://etherscan.io/address/0xeBcE83e491947aDB1396Ee7E55d3c81414fB0D47) |

> | PBG         | 3,555         | [0x8D4df847dB7FfE0B46AF084fE031F7691C6478c2](https://etherscan.io/address/0x8D4df847dB7FfE0B46AF084fE031F7691C6478c2) |

Reviewer 3: ```suggestion
| PBG         | 3,355         | [0x8D4df847dB7FfE0B46AF084fE031F7691C6478c2](https://etherscan.io/address/0x8D4df847dB7FfE0B46AF084fE031F7691C6478c2) |
```

### Spark Proxy Spell

If this executive proposal passes, then a Spark Proxy Spell will be executed at [0xA8FF99Ac98Fc0C3322F639a9591257518514455c](https://etherscan.io/address/0xA8FF99Ac98Fc0C3322F639a9591257518514455c) with the following contents.

#### [Arbitrum] Spark Liquidity Layer - Onboard Fluid sUSDS

- **Authorization**: [Governance Poll 1202](https://vote.makerdao.com/polling/QmZbk3gD)
- **Proposal**: [Forum Post](https://forum.sky.money/t/april-17-2025-proposed-changes-to-spark-for-upcoming-spell/26234)

Onboard Fluid sUSDS onto the Spark Liquidity Layer on Arbitrum with the following [Rate Limits](https://docs.spark.fi/dev/spark-liquidity-layer/spark-alm-controller#rate-limits):

- Deposits
  - `Max amount`: **10 million USDS worth of sUSDS**
  - `Slope`: **5 million USDS worth of sUSDS**
- Withdrawals
  - `Max amount`: **unlimited**

#### [Mainnet] Spark Liquidity Layer - Onboard SparkLend DAI

- **Authorization**: [Governance Poll 1224](https://vote.makerdao.com/polling/QmbNssss)
- **Proposal**: [Forum Post](https://forum.sky.money/t/april-17-2025-proposed-changes-to-spark-for-upcoming-spell/26234)

> Onboardi Mainnet SparkLend DAI to the Spark Liquidity Layer with the following parameters:

Reviewer 2: ```suggestion
Onboarding Mainnet SparkLend DAI to the Spark Liquidity Layer with the following parameters:
```

> Onboard Mainnet SparkLend DAI to the Spark Liquidity Layer with the following parameters:

Reviewer 2: ```suggestion
Onboard Mainnet SparkLend DAI to the Spark Liquidity Layer with the following [Rate Limits](https://docs.spark.fi/dev/spark-liquidity-layer/spark-alm-controller#rate-limits):
```

- Deposits
  - `Max amount`: **100 million DAI**
  - `Slope`: **50 million DAI per day**
- Withdrawals
  - `Max amount`: **unlimited**

#### [Mainnet] Spark Liquidity Layer - Onboard Curve sUSDS/USDT Pool

- **Authorization**: [Governance Poll 1223](https://vote.makerdao.com/polling/QmWgkXDA)
- **Proposal**: [Forum Post](https://forum.sky.money/t/april-17-2025-proposed-changes-to-spark-for-upcoming-spell/26234)

Onboard the Curve sUSDS/USDT pool to the Spark Liquidity Layer on Mainnet with the following parameters:

- Swap
  - `Max amount`: **5 million**
  - `Slope`: **20 million per day**
  - `Max slippage`: **0.15%**
- Deposits
  - `Max amount`: **5 million**
  - `Slope`: **20 million per day**
  - `Max slippage`: **0.15%**
- Withdrawals
  - `Max amount`: **25 million**
  - `Slope`: **100 million per day**
  - `Max slippage`: **0.15%**

#### [Mainnet] Spark Liquidity Layer - Onboard Curve USDC/USDT Pool

- **Authorization**: [Governance Poll 1222](https://vote.makerdao.com/polling/QmWju9Uu)
- **Proposal**: [Forum Post](https://forum.sky.money/t/april-17-2025-proposed-changes-to-spark-for-upcoming-spell/26234)

> Onboard the Curve USDC/USDT pool to the Spark Liquidity Layer on Mainnet with the following parameters:

Reviewer 2: ```suggestion
Onboard the Curve USDC/USDT pool to the Spark Liquidity Layer on Mainnet with the following [Rate Limits](https://docs.spark.fi/dev/spark-liquidity-layer/spark-alm-controller#rate-limits):
```

- Swap
  - `Max amount`: **5 million**
  - `Slope`: **20 million per day**
  - `Max slippage`: **0.15%**
- Deposit
  - `Max amount`: **0**
- Withdrawals
  - `Max amount`: **0**

#### [Mainnet] SparkLend - Adjust cbBTC and tBTC Interest Rate Models

- **Authorization**: [Governance Poll 1227](https://vote.makerdao.com/polling/Qmc3WXej)
- **Proposal**: [Forum Post](https://forum.sky.money/t/april-17-2025-proposed-changes-to-spark-for-upcoming-spell/26234)

Adjust the cbBTC and tBTC [Interest Rate Models](https://github.com/makerdao/next-gen-atlas/blob/7cf37209d2c32b88fc9dbb2017dca30b02b66a44/Sky%20Atlas/Sky%20Atlas.html#L8525) as follows:

- [Optimal utilization ratio](https://github.com/makerdao/next-gen-atlas/blob/7cf37209d2c32b88fc9dbb2017dca30b02b66a44/Sky%20Atlas/Sky%20Atlas.html#L8620): Increase by 20 percentage points from 60% to **80%**.
- [Base rate](https://github.com/makerdao/next-gen-atlas/blob/7cf37209d2c32b88fc9dbb2017dca30b02b66a44/Sky%20Atlas/Sky%20Atlas.html#L8645): Keep unchanged at **0%**.

> - [Slope 1](https://github.com/makerdao/next-gen-atlas/blob/7cf37209d2c32b88fc9dbb2017dca30b02b66a44/Sky%20Atlas/Sky%20Atlas.html#L8667): Decrease by 3 percentage points from 4% to **1%**.

Reviewer 3: ```suggestion
- [Slope 1](https://sky-atlas.powerhouse.io/A.AG1.3.2.2.1.1.15_Slope_1_Definition/1c1f2ff0-8d73-81be-9f17-d7d4bfc6cb59|7896ed3326389fe3553030cd0a82f68efd49): Decrease by 3 percentage points from 4% to **1%**.
```

> - [Slope 2](https://github.com/makerdao/next-gen-atlas/blob/7cf37209d2c32b88fc9dbb2017dca30b02b66a44/Sky%20Atlas/Sky%20Atlas.html#L8687): Keep unchanged at **300%**.

Reviewer 3: ```suggestion
- [Slope 2](https://sky-atlas.powerhouse.io/A.AG1.3.2.2.1.1.17_Slope_2_Definition/1c1f2ff0-8d73-8199-aa99-dde95c3f7130|7896ed3326389fe3553030cd0a82f68efd49): Keep unchanged at **300%**.
```

#### [Mainnet] SparkLend - Reduce WBTC LT

- **Authorization**: [Governance Poll 1229](https://vote.makerdao.com/polling/QmU3y9jf)
- **Proposal**: [Forum Post](https://forum.sky.money/t/april-17-2025-proposed-changes-to-spark-for-upcoming-spell/26234)

> Reduce the WBTC Liquidation Threshold as follows:

Reviewer: ```suggestion
Reduce the WBTC [Liquidation Threshold](https://github.com/makerdao/next-gen-atlas/blob/7cf37209d2c32b88fc9dbb2017dca30b02b66a44/Sky%20Atlas/Sky%20Atlas.html#L8560) as follows:
```

Reviewer 3: ```suggestion
Reduce the WBTC [Liquidation Threshold](https://sky-atlas.powerhouse.io/A.AG1.3.2.2.1.1.5_Liquidation_Threshold_Definition/1c1f2ff0-8d73-81ea-bd7e-feb73a44923a%7C7896ed3326389fe3553030cd0a82f68efd49) as follows:
```

- Reduce WBTC Liquidation Threshold by 5 percentage points from 50% to **45%**.

#### [Mainnet] SparkLend - Add sUSDS to USD Emode

- **Authorization**: [Governance Poll 1226](https://vote.makerdao.com/polling/QmQk4XKy)
- **Proposal**: [Forum Post](https://forum.sky.money/t/april-17-2025-proposed-changes-to-spark-for-upcoming-spell/26234)

Add sUSDS to the USD-correlated [E-mode](https://github.com/makerdao/next-gen-atlas/blob/7cf37209d2c32b88fc9dbb2017dca30b02b66a44/Sky%20Atlas/Sky%20Atlas.html#L8568).

#### [Mainnet] Morpho DAI Vault - Onboard July sUSDe PT

- **Authorization**: [Governance Poll 1228](https://vote.makerdao.com/polling/QmQgnWbX)

> - **Proposal**: [Forum Post](https://forum.sky.money/t/march-6-2025-proposed-changes-to-spark-for-upcoming-spell/26036)

Reviewer: ```suggestion
- **Proposal**: [Forum Post](https://forum.sky.money/t/april-17-2025-proposed-changes-to-spark-for-upcoming-spell/26234)
```

> Add sUSDS to the USD-correlated [E-mode](https://github.com/makerdao/next-gen-atlas/blob/7cf37209d2c32b88fc9dbb2017dca30b02b66a44/Sky%20Atlas/Sky%20Atlas.html#L8568).

Reviewer 3: ```suggestion
Add sUSDS to the USD-correlated [E-mode](https://sky-atlas.powerhouse.io/A.AG1.3.2.2.1.1.6_High_Efficiency_Mode_Category_Definition/1c1f2ff0-8d73-815c-8a8f-f3a095137e06|7896ed3326389fe3553030cd0a82f68efd49).
```

Onboard PT-sUSDe-31July2025 to the Morpho DAI vault with the following parameters:

- Pool LLTV: **91.5%**
- Pendle oracle discount rate: **20% APY**
- Supply cap: **400 million DAI**

#### [Mainnet, Base, Arbitrum] Spark Liquidity Layer - Upgrade Controller to v1.4.0 on All Chains

- **Authorization**: [Governance Poll 1225](https://vote.makerdao.com/polling/QmRkQDMT)

> - **Proposal**: [Forum Post](https://forum.sky.money/t/march-6-2025-proposed-changes-to-spark-for-upcoming-spell/26036)

Reviewer: ```suggestion
- **Proposal**: [Forum Post](https://forum.sky.money/t/april-17-2025-proposed-changes-to-spark-for-upcoming-spell/26234)
```
